### PR TITLE
Interface file doc

### PR DESCRIPTION
**Pull request by alex-tpom6oh**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**/implementation/cli**](https://docs-454s06knd.avo-dev.app/implementation/cli) |
|:-:|
|<a href="https://docs-454s06knd.avo-dev.app/implementation/cli">Screenshot of /implementation/cli</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/dlRwjNAOVHXeEs0CGQ-cNghYiM6MYM-_u-vJS4ltnotbMH5jZz5VCXxmzyF8RsI_kBgZznvS1v72Ko5yY-BWgw.png">(view full size)</a> |







Commit 3148de6914a8b9e9f4fcd1ec4fd340bc55fadf6a (https://docs-454s06knd.avo-dev.app).

File: pages/implementation/cli.mdx [hunk 1]
```diff
@@ -106,6 +108,28 @@ info Looking for events in src/main/java/app/avo/musicplayerexample/Player.kt
...
 
+### Interface files
+
+Some languages allow you to generate multiple files, for example Objective-C requires it and you can enable it in Kotlin and Swift using on-demand feature flags. <Link passHref href="/cli#step-3-pull-generated-analytics-wrappers-from-avo" passHref><a>see --forceFeatures pull flag above</a></Link>
+
+When initializing a source with CLI v3.2.0 and later it will ask you to specify a separate path for the interface file.
+
+You can also specify the interface file path manually, by adding `interfaceFilePath` field to the `source` object in the `avo.json` file
+
+```
```
Comment (alex-tpom6oh) [inline pages/implementation/cli.mdx:119]:
```suggestion
```bash
```

**Review by hugihlynsson — APPROVED** [review]
LGTM!

**Review by hugihlynsson — COMMENTED** [review]
LGTM

**Pull request merged**